charts/ks-devops/charts/s2i/values.yaml

```diff
@@ -2,9 +2,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-docker:
-  registry: ""
-  group: "kubespheredev"
+image:
+  # docker.io/kubespheredev, ghcr.io/kubespheredev or any other registries
```

Comment:
> or any other registries

Really? IMO it's not accurate. How about `gcr.io`?

Reply:
@JohnNiang How about this one?

Reply:
`gcr.io` is not supported currently. However, my original intention is private registry. I will change it according README.md.

Reply:
I can guess what do you want to address here. But we need to make it be accurate.

Reply:
Alright.